File: src_a_(C++)/Test1-7.c
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 1:СТРУКТУРЫ ДАННЫХ
//Тест номер    7:НОВЫЙ ТИП

include Win32

char s[15];

define con 0x0009
typedef cardinal new;

new i;

void main() {
  i=(new)con;
  wvsprintf(s,"i=%li",&i);
  MessageBox(0,s,"i=9",0);
}

File: src_a_(Pascal)/Test1-7.pas
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 1:СТРУКТУРЫ ДАННЫХ
//Тест номер    7:НОВЫЙ ТИП
program Test1_7;
uses Win32;

var s:string[15];

const con=0x0009;
type new=cardinal;

var i:new;

begin
  i:=new(con);
  wvsprintf(s,"i=%li",addr(i));
  MessageBox(0,s,"i=9",0);
end.

